examples/time-log-kopia-repository-server/README.md

```diff
@@ -0,0 +1,233 @@
+# Walkthrough of Time Log
```

Comment:
```suggestion
# Protecting TimeLog application
```

Reply:
I think we should use TimeLog instead of `Time Log` and if we agree on that let's change that everywhere else.